tutorials/migration_sdf.md

```diff
@@ -0,0 +1,336 @@
+            <time>0</time>
+            <pose>2 -2 0.5 0 0 0</pose>
+          </waypoint>
+          <waypoint>
+            <time>5</time>
+            <pose>2 -2 0.5 0 0 0</pose>
+          </waypoint>
+        </trajectory>
+      </script>
+    </actor>
+
+  </world>
+</sdf>
+```
+
+\note The actor's vertical pose will be different on both simulators.
+      That's because a hardcoded offset was removed on Ignition and
+      maintained on Gazebo classic for backwards compatibility.
+
+### Path relative to the SDF file
+
+It's possible to use relative paths within SDF files to refer to other files
+in the same directory. This is recommended when creating files that work
+together and need to be relocatable.
+
+For example, consider the following directory structure:
+
+```
+/home/username/
+├── world.sdf
+└── models
+    └── model1
+        ├── model.sdf
+        └── meshes
+            └── mesh.dae
+```
+
+The world `world.sdf` can include `model1` as follows:
+
+`<include>models/model1</include>`
+
+And `model.sdf` can refer to `mesh.dae` as:
+
+`<uri>meshes/mesh.dae</uri>`
+
+### Path relative to an environment variable
+
+This method is useful if you don't know where the files will be located
+with respect to each other, and you have some control over the simulation's
+runtime environment. On either simulator, you can refer to resources relative
+to paths set on an environment variable.
+
+Each simulator uses a different environment variable:
+
+* Gazebo classic:
+    * `GAZEBO_MODEL_PATH` for models
+    * `GAZEBO_RESOURCE_PATH` for worlds and some rendering resources
+* Ignition Gazebo:
+    * `IGN_GAZEBO_RESOURCE_PATH` for worlds, models and other resources
+
+For example, if you have the file structure above, you can set the environment
+variable to `/home/username/models`:
+
+```
+export GAZEBO_MODEL_PATH=/home/username/models
+export GAZEBO_RESOURCE_PATH=/home/username/models
+export IGN_GAZEBO_RESOURCE_PATH=/home/username/models
+```
+
+And inside `world.sdf` include the model with:
+
+`<include>model://model1</include>`
+
+Also, `model.sdf` can refer to `mesh.dae` as:
+
+`<uri>model://model1/meshes/mesh.dae</uri>`
+
+On both situations, the `model://` prefix will be substituted by
+`/home/username/models`.
+
+You can also set several lookup paths separating them with `:`, for example:
+
+`export IGN_GAZEBO_RESOURCE_PATH=/home/username/models:/home/username/another_project/models`
+
+### Absolute paths
+
+Finally, both simulators will accept absolute paths, for example:
+
+`<include>/home/username/models/model1</include>`
+
+This method is not recommended, because the file most likely won't work if
+copied to a different computer. It also becomes inconvenient to move files to
+different directories.
+
+## Plugins
+
+Plugins are binary files compiled to use with a specific simulator. Plugins
+for Gazebo classic and Ignition Gazebo aren't compatible, so plugins will need
+to be specified for each simulator separately.
```

Comment:
It may be important to note that plugins from different versions (citadel vs dome) are also not likely to be compatible.

Reply:
Oh I see what you mean. From the SDF file's point of view, nothing should change though. I'll clarify.

Reply:
See fc5db21